doc/whats_new/v0.21.rst

```diff
@@ -87,6 +87,10 @@ Support for Python 3.4 and below has been officially dropped.
   :class:`linear_model.MultiTaskLasso` which were breaking when
   ``warm_start = True``. :issue:`12360` by :user:`Aakanksha Joshi <joaak>`.
 
+- |API| :func:`linear_model.logistic_regression_path` is deprecated
```

Comment:
I think usually we put this into the "deprecation" section that should be below. otherwise lgtm

Reply:
Are you sure? I see a lot of `|API| xxx has been deprecated` in every sub-section in `0.20.rst`

Reply:
oh sorry I guess we changed that.